modules/govuk_jenkins/templates/jobs/govuk_tagging_monitor.yaml.erb

```diff
@@ -0,0 +1,30 @@
+---
+- scm:
+    name: govuk_tagging_monitor
+    scm:
+      - git:
+          url: git@github.com:alphagov/govuk-tagging-monitor.git
+          basedir: govuk-tagging-monitor
+          branches:
+            - master
+- job:
+    name: govuk_tagging_monitor
+    display-name: GOV.UK Tagging Monitor
+    project-type: freestyle
+    description: |
+      This job checks new navigation pages against a set of rules. This makes
+      sure we don't have invalid navigation pages being shown to users.
+    scm:
+      - govuk-tagging-monitor
+    logrotate:
+      numToKeep: 10
+    triggers:
+        - timed: '35 * * * *'
```

Comment:
I'm not sure I understand the question. I copied this from the existing job following some of the existing patterns: https://github.com/alphagov/govuk-puppet/blob/a6c51d887a6501f02766b7279127b60f02037a7f/modules/govuk_jenkins/templates/jobs/smokey_deploy.yaml.erb#L24

Would you like this to be done differently?